tests/lib/session.js

```diff
@@ -72,6 +72,41 @@ define([
         );
       });
 
+      test('#sessions', function () {
+
+        return accountHelper.newVerifiedAccount()
+          .then(function (account) {
+            return respond(client.sessions(account.signIn.sessionToken), RequestMocks.sessions);
+          })
+          .then(
+            function (res) {
+              assert.equal(res.length, 2);
+              assert.ok(res[0].id);
+              assert.ok(res[0].userAgent);
+              assert.ok(res[0].deviceName);
```

Comment:
Noticed some other values that might be useful to verify to assert in the response, `isDevice=false`, `deviceType=desktop`.

![screen shot 2017-02-15 at 5 37 33 pm](https://cloud.githubusercontent.com/assets/1295288/22998710/f4c4a06a-f3a5-11e6-8058-0d9e69d8aacd.png)